src/dash-sai/Makefile

```diff
@@ -0,0 +1,57 @@
+SHELL = /bin/bash
+.ONESHELL:
+.SHELLFLAGS += -xe
+
+MAIN_TARGET = libsai_$(DASH_SAI_VERSION)_$(CONFIGURED_ARCH).deb
+DERIVED_TARGETS = libsai-dev_$(DASH_SAI_VERSION)_$(CONFIGURED_ARCH).deb
+
+CURRENT_DIR=$(shell pwd)
+PIPELINE_DIR=$(CURRENT_DIR)/DASH/dash-pipeline
+P4_MAIN=bmv2/dash_pipeline.p4
+P4_OUTDIR=bmv2/dash_pipeline.bmv2
+
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+# Remove any stale files
+	rm -rf DASH
+
+	git clone https://github.com/sonic-net/DASH.git DASH
```

Comment:
is this how we workaround the triangle submodule references on SAI repo?

Reply:
we might need to sync on this approach.

Reply:
I don't want to use submodule because the DASH repo may fail due to some stale files from the last building. It's not reasonable to clean up a submodule in this Makefile.